chcemvediet/apps/inforequests/templates/inforequests/detail/texts/deadline.html

```diff
@@ -25,15 +25,8 @@
   na podanie odvolania
 {% endif %}
 &mdash;
-{% if action.deadline.is_deadline_missed %}
-  <span class="chv-color-red">
-    Lehota uplynula {{ action.deadline.deadline_date|date:"j. E Y" }}.
-  </span>
-{% else %}
-  <span class="chv-color-green">
-    Lehota uplynie {{ action.deadline.deadline_date|date:"j. E Y" }}.
-  </span>
-{% endif %}
```

Comment:
Texts templaty by nemali dalej includovat dalsie templaty. Mali by to byt uz konecne templaty iba s textom, bez dalsej logiky. Tiez by sa nemali includovat napriec adresarmi. Nech text templaty mame hned vedla templatov, co ich pouzivaju.

Navyse v tabulke chceme zobrazit text "Lehota **vám/inštitúcii** uplynie o X dni", kdezto tu nespominame, ci letoha plynie pre usera alebo pre instituciu, kedze z kontextu je to jasne. Taktiez tu cervernu/zelenu farbu v tabulke asi nechceme.

Sprav teda radsej samostatny text template pre deadline v `inforequests/tables/texts/` nezavisli na tomto text template.